### DISCUSSION TITLE
'pytorch_lightning.core' has no attribute 'decorators'

### DISCUSSION THREAD

**Discussion by liw17015** [Lightning-AI/pytorch-lightning#12840] [General]
Hi, I am using Cudatoolkit 11.3 and PyTorch 11.1. My code worked for pytorch_lightning 1.3.4, but after  I changed to pytorch_lighning 1.6.1, It shows:

module 'pytorch_lightning.core' has no attribute 'decorators'

Does it mean the 1.6.1 version no longer supports decorators? what should I do if I don't wanna change my code?

Thanks!
![image](https://user-images.githubusercontent.com/63358207/164542459-777698fe-89a1-4a99-b309-fd55ffdcc240.png)

**Comment by akihironitta** [comment]
Hi @liw17015! The decorator `pytorch_lightning.core.decorators.auto_move_data` was deprecated in v1.3 and removed in v1.5, so I'd try either using `pytorch-lightning<1.5` or updating the code so as not to use the decorator. Reference: #6993
